### DISCUSSION TITLE
Error updating datasource schema : No values in the header row - fill the first row with header values before trying to interact with rows

### DISCUSSION THREAD

**Discussion by blocka** [Budibase/budibase#9366] [Help]
I'm trying to connect with google sheets. Under "Fetch Tables" I entered the sheet I want to sync.
I get the above error despite that fact that there is a header row.

**Comment by andz-bb** [comment]
hey @blocka,

I haven't been able to recreate this on my end. Could you send an example sheet where this isn't working please? Thanks

**Comment by blocka** [comment]
I reproduced it with this sheet: https://docs.google.com/spreadsheets/d/1e-NcAv5MQs4_KAS0oIYTT1EXbHrSAfrdRZ0Vo_mRArs/edit?usp=sharing

**Comment by andz-bb** [comment]
Thanks for sending that example sheet @blocka 

Whenever I added a few rows with fake data my Budibase was able to parse it out into a table properly.

![google sheets example](https://user-images.githubusercontent.com/110921612/213707280-18474dd3-fbe6-411f-b3ee-8d7e89781d4d.png)

Could you add a row or two to your example sheet so that I can check if there's a problem with the format of it? The example rows I added to a copy of your sheet worked OK but I maybe got the datatypes wrong in my test data.

**Comment by martinrosbier** [comment]
Hi everyone, I'm having the same issue here, tried adding more rows to the sheet but same problem. How to force to detect the first row as a header? any work around?
